hc/accounts/tests/test_switch_team.py

```diff
@@ -22,10 +23,12 @@ def test_it_checks_team_membership(self):
         url = "/accounts/switch_team/%s/" % self.alice.username
```

Comment:
Check out string formatting with python. Typically we prefer to use `.format`
i.e.
`"/accounts/switch_team/{}/".format(self.alice.username)`